docs/examples/shading/plot_simple_irradiance_adjustment_for_horizon_shading.py

```diff
@@ -0,0 +1,108 @@
+"""
+Simple Irradiance Adjustment For Horizon Shading
+================================================
+
+Example for adjusting dni and global poa from horizon shading. Using
+hard-coded horizon profile data, this example interpolates it to time-series
+solar-position data, and adjust DNI and POA-global irradiance.
+"""
+
+# %%
+# This example shows how to use horizon elevation angles with
+# corresponding horizon azimuth angles for simple horizon shading adjustments.
+
```

Comment:
```suggestion
#
```

Connecting these two comment blocks will make the second one render as HTML text instead of code